### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Create a database named vet_clinic.
- [x] Create a table named animals with the required columns.
- [x] Insert the required data.
- [x] Create required queries.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES(1, 'Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals VALUES(2, 'Gabumon', '2018-11-18', 2, true, 8);
+INSERT INTO animals VALUES(3, 'Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals VALUES(4, 'Devimon', '2017-05-12', 5, true, 11);
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```
Comment (Meltrust) [inline data.sql:10]: - [ ] These queries should be placed on `queries.sql`, not on `data.sql`.  Kindly remove them from this file and copy them into `schema.sql`.  That way, your code will be well organized and we will meet this requirement 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @fabgrel10,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice readme ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES(1, 'Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals VALUES(2, 'Gabumon', '2018-11-18', 2, true, 8);
```
[the remark this answers is not in the document]
Comment (fabgrel10) [reply data.sql]: Fixed that one

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```
[the remark this answers is not in the document]
Comment (fabgrel10) [reply queries.sql:4]: Fixed that one

[the remark this answers is not in the document]
Comment (fabgrel10) [reply queries.sql]: Fixed this one

File: data.sql (hunk 1 above)
Comment (fabgrel10) [reply data.sql]: Fixed that one.

**Review by julie-ify — APPROVED** [review]
Hi @fabgrel10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabgrel10** [comment]
Thank you for the review @Meltrust and @julie-ify!

**Pull request merged**